packages/wonder-blocks-dropdown/components/dropdown.test.js

```diff
@@ -257,5 +257,63 @@ describe("Dropdown", () => {
             // Assert
             expect(controlledComponent.state("selectedValues")).toBeFalsy();
         });
+
+        it("only single selects OptionItems if singleSelectOption is passed", () => {
+            // Arrange
+            const controlledComponent = mount(
+                <ControlledComponent singleSelectOption />,
+            );
+            controlledComponent.simulate("click");
+            controlledComponent.setState({selectedValues: ["B"]});
+            const dropdownCore = controlledComponent.find(DropdownCore);
+
+            // Act
+            dropdownCore.simulate("keydown", {keyCode: keyCodes.down});
+            dropdownCore.simulate("keyup", {keyCode: keyCodes.down});
+
+            const optionItem = controlledComponent.find(OptionItem).at(0);
+            optionItem.simulate("click");
+
+            // Assert
+            expect(controlledComponent.state("selectedValues")).toHaveLength(1);
+        });
+
+        it("closes Dropdown if an OptionItem is selected when singleSelectOption is passed", () => {
+            // Arrange
+            const controlledComponent = mount(
+                <ControlledComponent singleSelectOption />,
+            );
+            controlledComponent.simulate("click");
+            controlledComponent.setState({selectedValues: ["B"]});
+            const dropdownCore = controlledComponent.find(DropdownCore);
+
+            // Act
+            dropdownCore.simulate("keydown", {keyCode: keyCodes.down});
+            dropdownCore.simulate("keyup", {keyCode: keyCodes.down});
```

Comment:
This should probably be part of `// Arrange` since the thing that's being tested here is what happens when an option is selected.